### PR TITLE
Add OpenStack cloud with ppc64le compute nodes inside Red Hat for 4.3 CI testing

### PR DESCRIPTION
**Pull request by hamzy**
Add OpenStack cloud with ppc64le compute nodes inside Red Hat for 4.3 CI testing

### CODE REVIEW

File: ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml [hunk 1]
```diff
@@ -0,0 +1,731 @@
...
+        fi
+
+        # set up env vars
+        export KUBE_SSH_BASTION="$( oc --insecure-skip-tls-verify get node -l node-role.kubernetes.io/master -o 'jsonpath={.items[0].status.addresses[?(@.type=="ExternalIP")].address}' ):22"
+        export KUBE_SSH_KEY_PATH=/tmp/cluster/ssh-privatekey
+        export KUBE_SSH_USER=core
+        mkdir -p ~/.ssh
+        cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_openstack_rsa || true
+
+        export OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE=https://art-rhcos-ci.s3.amazonaws.com/releases/rhcos-4.3-ppc64le/43.81.202001081114.0/ppc64le/rhcos-43.81.202001081114.0-openstack.ppc64le.qcow2.gz
```
Comment (crawford) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml:170]: Why is this needed? Shouldn't https://github.com/openshift/installer/pull/3233 have provided this information?

Comment (hamzy) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml:170]: This patch was started long before that commit.  Let me try it without the override...

Comment (hamzy) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml]: Nope, still need the override.  I am seeing:
`level=info msg="Obtaining RHCOS image file from 'https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.202003111353.0/x86_64/rhcos-43.81.202003111353.0-openstack.x86_64.qcow2.gz?sha256=4d204e638d365d9de121f5d513cff2567abd9232710f4bb79992efa4ba718008'"
`

Comment (jaypoulz) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml]: @hamzy does this mean there is a problem with the baked in rhcos image?
Maybe @Prashanth684 can help?

Comment (hamzy) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml]: Keep in mind that the CI pod runs on amd64 while the OpenShift cluster is a ppc64le cluster.

Comment (Prashanth684) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml]: is `architecture` specified in the install-config.yaml for the master and computes? if not it's going to pick x86

File: ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml [hunk 2]
```diff
@@ -0,0 +1,726 @@
...
+
+        echo "Creating DNS record for api.${CLUSTER_NAME}.${BASE_DOMAIN} -> ${LB_FIP}"
+        # create A record for the api
+        curl -v http://sshd.bastion-ppc64le:13053/add -d '{"cluster_name": "'${CLUSTER_NAME}.${BASE_DOMAIN}'", "lb_fip": "'$LB_FIP'"}'
+
+        cat > /tmp/artifacts/installer/install-config.yaml << EOF
+        apiVersion: v1beta4
+        baseDomain: ${BASE_DOMAIN}
+        controlPlane:
+          hyperthreading: Enabled
```
Comment (Prashanth684) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml:392]: `architecture: ppc64le` needs to be specified if not it will pick an x86 image

File: ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml (hunk 1 above)
Comment (hamzy) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml]: Thanks Prashanth for pointing out the needed architecture keys in install-config.yaml

File: core-services/prow/02_config/_plugins.yaml [hunk 3]
```diff
@@ -1080,6 +1080,17 @@ config_updater:
         build01:
         - ci
       name: prow-job-cluster-launch-installer-nested-virt-tests
+    ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml:
+      clusters:
+        api.ci:
+        - ci
+        - ci-stg
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:1087]:
```suggestion
```

File: core-services/prow/02_config/_plugins.yaml [hunk 4]
```diff
@@ -1080,6 +1080,17 @@ config_updater:
...
+        app.ci:
+        - ci
+        - ci-stg
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:1090]:
```suggestion
```

File: core-services/prow/02_config/_plugins.yaml [hunk 5]
```diff
@@ -1080,6 +1080,17 @@ config_updater:
...
+        ci/api-build01-ci-devcluster-openshift-com:6443:
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:1091]:
```suggestion
        build01:
```
